microprofile/reactive-messaging-smallrye/common/src/main/java/org/wildfly/microprofile/reactive/messaging/common/_private/MicroProfileReactiveMessagingCommonLogger.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * JBoss, Home of Professional Open Source.
```

Comment:
This should be ASL now?

Reply:
See L24-27 -- it looks like the header switch got messed up.

Reply:
@bstansberry already fixed?